modules/jre/8/module.yaml

```diff
@@ -14,7 +14,7 @@ labels:
 
 envs:
 - name: "JAVA_HOME"
-  value: "/usr/lib/jvm/jre"
+  value: "/usr/lib/jvm"
```

Comment:
We can't set `JAVA_HOME` to this value here, because it mean `$JAVA_HOME/bin/java` would not exist. Instead, we need to adjust the code you identified at https://github.com/jboss-container-images/openjdk/blob/develop/modules/run/configure.sh#L20 to handle the discrepancy between correct paths for the runtime and jdk images for 8u, probably as a third logic branch (`elif`)  in between the existing two.